memcached/content.md

```diff
@@ -12,12 +12,31 @@ Memcached's APIs provide a very large hash table distributed across multiple mac
 $ docker run --name my-memcache -d %%IMAGE%%
 ```
 
-## Setting Memory Usage
+### Configuration
+
+To customize the configuration of the memcached server, first obtain the upstream configuration options from the container:
+
+```console
+$ docker run --rm %%IMAGE%% -h
+```
+
+Using `docker run`:
 
 ```console
-$ docker run --name my-memcache -d %%IMAGE%% memcached -m 64
+docker run --name my-memcache -d %%IMAGE%% memcached:alpine --memory-limit=64
```

Comment:
This instance of `memcached` was the command to run, not the image name :sweat_smile:

I also don't see a good reason to prefer `:alpine`, especially as we have a section below that explains why Alpine is perhaps not ideal (I don't think they've explicitly stated it, but I'd be very surprised if upstream considers Alpine / musl as an "officially supported" target :see_no_evil:)

```suggestion
$ docker run --name my-memcache -d %%IMAGE%% --memory-limit=64
```